### DISCUSSION TITLE
Use rich text editor outside of Keystone

### DISCUSSION THREAD

**Discussion by leonard-henriquez** [keystonejs/keystone#7551] [Questions] [answered]
Hi,

## Context

I have a Next.js + Nest.js application that doesn't use Keystone.
I fell in love with your rich text editor, but I the rest of Keystone doesn't really fit my needs.
I _**could**_ bend Keystone into my use case but it would add too much complexity to be worth it.

## Questions
- Is it possible to use the rich text editor and the renderer outside of Keystone?
- Do you have an example how to do that?
- Do you plan on documenting the packages `@keystone-6/fields-document` and `@keystone-6/document-renderer`?

## Note for the team
Thank you very much for your work ! Your project is absolutely awesome 🤩.
Just a friendly advice, I think you could increase adoption by allowing to use only some parts of your project alone.

**Comment by raveling** [comment, marked as answer]
@leonard-henriquez you can't currently achieve this. We’re tracking this at https://github.com/keystonejs/keystone/discussions/5488

**Reply by leonard-henriquez** [reply]
What do you mean by I can't acheive it?
In the thread that you linked there are at least two working examples of using the rich text editor outside Keystone
But unfortunately they are undocumented...

**Comment by yangricardo** [comment]
I am in look for something similar, as i have an application that is supported by KeystoneJS, but the form to final user should be not be used with KeystoneJS UI, indeed, i have a field that should be completed by using an Rich Text Editor. Today i use the TinyMCE, but i am looking to change to SlateJS (https://www.slatejs.org/examples/richtext), the same used by KeystoneJS, in my frontend application. At database layer, i'd recommend you to use a JSON type column...
